src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -22,8 +22,6 @@ public class NamespaceCache {
 
   private final ObjectMapper objectMapper;
 
-  private final boolean apiTablesEnabled;
```

Comment:
Had to remove Tables-enabled feature from here because we have to be able enable/disable it on per-request basis. So we will always load regular Table schema find, and fail request at a later point.
Seems more correct overall anyway; checking here would be too low level.

Reply:
sounds reasonable